controllers/awsmachine_controller_test.go

```diff
@@ -60,9 +61,16 @@ var _ = Describe("AWSMachineReconciler", func() {
 		klog.SetOutput(GinkgoWriter)
 		var err error
 
+		awsmachine := &infrav1.AWSMachine{
+			ObjectMeta: metav1.ObjectMeta{
+				Name: "",
+			},
+			Spec: infrav1.AWSMachineSpec{},
+		}
+
```

Comment:
I can't comment on line 79/87, but it would probably be worthwhile to use this `awsMachine` as the value for `AWSMachine` in `MachineScopeParams`.

Reply:
agreed ... updated.